test/e2e/framework/autoscaling/autoscaling_utils.go

```diff
@@ -29,6 +29,7 @@ import (
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	crdclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/apiextensions-apiserver/test/integration/fixtures"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
> ERROR wrong alias for import "k8s.io/apimachinery/pkg/api/errors" should be apierrors in file test/e2e/framework/autoscaling/autoscaling_utils.go

/hold
feel free to remove the hold once this is fixed.

```suggestion
	apierrors "k8s.io/apimachinery/pkg/api/errors"
```